src/app/containers/App/index.jsx

```diff
@@ -6,11 +6,14 @@
 import React from 'react';
 import { StaticRouter, BrowserRouter } from 'react-router-dom';
 import App from './App';
+import { EventContextProvider } from '../../contexts/EventContext'
 
 export const ClientApp = props => (
-  <BrowserRouter {...props}>
-    <App initialData={props.data} routes={props.routes} />
-  </BrowserRouter>
+  <EventContextProvider>
```

Comment:
We can move `<EventContextProvider>` to `simorgh/src/app/containers/PageHandlers/withContexts.jsx` :)

Reply:
Done!

Reply:
Its still here 🙈